File: ruby/yapay_gw/lib/yapay_gw/models/transaction_checkout_multiple_bill_data.rb
class TransactionCheckoutMultipleBillData
    attr_accessor :value
    attr_accessor :due_data
    
    def as_json(options = {})
        json = {:valor => value, 
                :vencimento => due_data
        }
        json.delete_if { |k, v| v.nil? }
        json
    end
end

File: dotnet/YapayGatewayLib/model/TransactionCheckoutMultipleBillData.cs
using System;
using Newtonsoft.Json;

namespace YapayGatewayLib.Model
{
    public class TransactionCheckoutMultipleBillData
    {
        [JsonProperty(PropertyName = "valor")]
        public long value { get; set; }

        [JsonProperty(PropertyName = "vencimento")]
        public string dueDate { get; set; }
    }
}
